pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -658,13 +659,26 @@ def status(self):
             return _("Local media via SE/HMC")
         elif self.data.method.method == "harddrive":
             if not self._currentIsoFile:
-                return _("Error setting up ISO file")
+                if self.payload.baseRepo:
+                    return "%s:%s" % (self._get_harddrive_partition_name(),
+                                      self.data.method.dir)
+                return _("Error setting up installation from HDD")
             return os.path.basename(self._currentIsoFile)
         elif self.payload.baseRepo:
             return _("Closest mirror")
         else:
             return _("Nothing selected")
 
+    def _get_harddrive_partition_name(self):
+        devices = device_matches(self.data.method.partition)
+        if not devices:
+            log.warning("Device for method harddrive partition can't be found!")
+            return ""
+        elif len(devices) > 1:
+            log.warning("More than one device is found for method harddrive partition!")
```

Comment:
I am not sure if the term "method harddrive partition" makes sense.

Reply:
Did `harddrive method partition` sounds better?

Reply:
If there's internally something called `harddrive partition`, how about quotes or such?

`log.warning("Device for 'harddrive partition' method can't be found!")`

Reply:
What about `Device for installation from HDD can't be found!`? I guess I don't like the `method` there. It is an internal name.

Reply:
Thanks @VladimirSlavik and @poncovka for a great suggestions! Fixed.